src/core/Akka/Event/LoggingBus.cs

```diff
@@ -164,46 +173,68 @@ internal void StopDefaultLoggers(ActorSystem system)
                 Publish(new Debug(SimpleName(this), GetType(), $"Shutting down: {Logging.SimpleName(system.Settings.StdoutLogger)} started"));
             }
 
+            // Cancel all pending logger initialization tasks
+            _shutdownCts.Cancel(false);
```

Comment:
This is the mechanism to shutdown all pending `Ask` if they're still in progress even when the ActorSystem is shutting down.